packages/react-ui/components/RadioGroup/__tests__/RadioGroup.test.tsx

```diff
@@ -264,4 +264,26 @@ describe('<RadioGroup />', () => {
 
     expect(() => render(<RadioGroupGeneric />)).not.toThrow();
   });
+
+  it('should have correctly role', () => {
+    function RadioGroupGeneric() {
+      return (
+        <RadioGroup>
+          <Radio value={'str'} />
+        </RadioGroup>
+      );
+    }
+
+    render(<RadioGroupGeneric />);
```

Comment:
Этот код можно упростить:
```suggestion
      render(
        <RadioGroup>
          <Radio value={'str'} />
        </RadioGroup>
      )
```

Reply:
done https://github.com/skbkontur/retail-ui/pull/3096/files/51494acd9e32b46f25bc783a2005ceb07901fe58..cc9d083ec80800537653635574b15a69507624ab